<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FixShapeUI</class>
 <widget class="QMainWindow" name="FixShapeUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Fix Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="btn_makeDriver">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Make  Driver</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="45,55">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QListView" name="listView">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_refresh">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Driver Control  Rotate:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,45,45">
           <item>
            <widget class="QCheckBox" name="CBX_X">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="Spx_minX">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-90.000000000000000</double>
             </property>
             <property name="maximum">
              <double>90.000000000000000</double>
             </property>
             <property name="value">
              <double>-90.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="Spx_maxX">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-90.000000000000000</double>
             </property>
             <property name="maximum">
              <double>90.000000000000000</double>
             </property>
             <property name="value">
              <double>90.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="10,45,45">
           <item>
            <widget class="QCheckBox" name="CBX_Y">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="Spx_minY">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-90.000000000000000</double>
             </property>
             <property name="maximum">
              <double>90.000000000000000</double>
             </property>
             <property name="value">
              <double>-90.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="Spx_maxY">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-90.000000000000000</double>
             </property>
             <property name="maximum">
              <double>90.000000000000000</double>
             </property>
             <property name="value">
              <double>90.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="10,45,45">
           <item>
            <widget class="QCheckBox" name="CBX_Z">
             <property name="text">
              <string>Z</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="Spx_minZ">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-90.000000000000000</double>
             </property>
             <property name="maximum">
              <double>90.000000000000000</double>
             </property>
             <property name="value">
              <double>-90.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="Spx_maxZ">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-90.000000000000000</double>
             </property>
             <property name="maximum">
              <double>90.000000000000000</double>
             </property>
             <property name="value">
              <double>90.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_SetTestKey">
             <property name="text">
              <string>Set Test Key</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_delTestKey">
             <property name="text">
              <string>Del Test Key</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Driver Axis:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="rdn_Box" native="true">
           <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QRadioButton" name="rdn_A">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>-X+Y</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="rdn_B">
              <property name="text">
               <string>+Y</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QRadioButton" name="rdn_C">
              <property name="text">
               <string>+X+Y</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="rdn_D">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>-X</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QRadioButton" name="rdn_E">
              <property name="text">
               <string>+X</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="rdn_F">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>-X-Y</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QRadioButton" name="rdn_G">
              <property name="text">
               <string>-Y</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QRadioButton" name="rdn_H">
              <property name="text">
               <string>+X-Y</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>BlendShape attribute:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="9,1">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_inputBsattr">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>&lt; &lt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>BlendShape Value:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="25,75">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QDoubleSpinBox" name="DNFLD">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>22</height>
              </size>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="DNSLD">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_setDrivenkey">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Set   Driven  Key</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_EditData">
             <property name="text">
              <string>Edit Data</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CBX_X</sender>
   <signal>clicked(bool)</signal>
   <receiver>Spx_minX</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBX_Y</sender>
   <signal>clicked(bool)</signal>
   <receiver>Spx_minY</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBX_Z</sender>
   <signal>clicked(bool)</signal>
   <receiver>Spx_minZ</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBX_X</sender>
   <signal>clicked(bool)</signal>
   <receiver>Spx_maxX</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBX_Y</sender>
   <signal>clicked(bool)</signal>
   <receiver>Spx_maxY</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBX_Z</sender>
   <signal>clicked(bool)</signal>
   <receiver>Spx_maxZ</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
